Failing test: Security Solution Cypress.x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation·cy·ts - Timelines should show the different timeline states should show the different timeline states

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 150000ms: Expected <span.euiLoadingSpinner.css-1fsqg4c-euiLoadingSpinner-l> not to exist in the DOM, but it was continuously found.
    at Context.eval (webpack:///./e2e/investigations/timelines/creation.cy.ts:108:30)
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/98100#019e4adb-dd2c-4a48-bd56-b87a008277db)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation·cy·ts","test.name":"Timelines should show the different timeline states should show the different timeline states","test.failCount":2}} -->

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation.cy.ts
@@ -102,9 +102,8 @@ describe('Timelines', { tags: ['@ess', '@serverless'] }, (): void => {
     // Saved
     cy.get(TIMELINE_STATUS).should('not.exist');
 
-    // Offsetting the extra save that is happening in the background
-    // for the saved search object.
-    cy.get(LOADING_INDICATOR).should('be.visible');
+    // Wait for any background saves (e.g. saved search object) to settle
+    // before typing in the KQL bar to avoid interfering with in-flight requests.
     cy.get(LOADING_INDICATOR).should('not.exist');
 
     executeTimelineKQL('agent.name : *');
@@ -124,9 +123,7 @@ describe('Timelines', { tags: ['@ess', '@serverless'] }, (): void => {
     openTimelineUsingToggle();
     addNameToTimelineAndSave('First');
 
-    // Offsetting the extra save that is happening in the background
-    // for the saved search object.
-    cy.get(LOADING_INDICATOR).should('be.visible');
+    // Wait for any background saves (e.g. saved search object) to settle.
     cy.get(LOADING_INDICATOR).should('not.exist');
 
     addNameToTimelineAndSaveAsNew('Second');

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation.cy.ts
@@ -19,7 +19,6 @@ import {
   SAVE_TIMELINE_ACTION_BTN,
   SAVE_TIMELINE_TOOLTIP,
 } from '../../../screens/timeline';
-import { LOADING_INDICATOR } from '../../../screens/security_header';
 import { ROWS } from '../../../screens/timelines';
 import { createTimelineTemplate, deleteTimelines } from '../../../tasks/api_calls/timelines';
 
@@ -102,10 +101,6 @@ describe('Timelines', { tags: ['@ess', '@serverless'] }, (): void => {
     // Saved
     cy.get(TIMELINE_STATUS).should('not.exist');
 
-    // Wait for any background saves (e.g. saved search object) to settle
-    // before typing in the KQL bar to avoid interfering with in-flight requests.
-    cy.get(LOADING_INDICATOR).should('not.exist');
-
     executeTimelineKQL('agent.name : *');
 
     // Saved but has unsaved changes
@@ -122,9 +117,6 @@ describe('Timelines', { tags: ['@ess', '@serverless'] }, (): void => {
 
     openTimelineUsingToggle();
     addNameToTimelineAndSave('First');
-
-    // Wait for any background saves (e.g. saved search object) to settle.
-    cy.get(LOADING_INDICATOR).should('not.exist');
 
     addNameToTimelineAndSaveAsNew('Second');
     closeTimeline();

